include/tvm/tir/op_attr_types.h

```diff
@@ -43,6 +45,11 @@ using TGlobalSymbol = String;
  */
 using TVectorizable = bool;
 
+/*!
+ * \brief The intrinsic lowering function for given OP.
+ */
+using FLowerIntrinsic = PackedFunc;
```

Comment:
Using `runtime::TypedPackedFunc` when possible, which is a simple wrapper of `PackedFunc`, but with the clear signature like `std::function`, so it could help readability :-)